.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+name: Test
+on:
+  push:
+    branches: [main]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [main]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Can you add the diff step so that it only runs when the js/ts files are updated?

Reply:
Yes, I'll add changes on packages.json and yarn.lock in case some dependency breaks the build/tests